plugins/win-dshow/cmake/libdshowcapture.cmake

```diff
@@ -0,0 +1,56 @@
+add_library(libdshowcapture INTERFACE)
```

Comment:
Here, because libdshowcapture's build system isn't designed to be pulled in as source files only. It's easier to create a "virtual" interface library for our purposes.

Reply:
Got it.